packages/types/src/json.ts

```diff
@@ -363,6 +364,21 @@ export interface OrganizationDomainJSON extends ClerkResourceJSON {
   updated_at: number;
 }
 
+export interface OrganizationSuggestionJSON extends ClerkResourceJSON {
+  object: 'organization_suggestion';
+  id: string;
+  public_organization_data: {
```

Comment:
Shall we create a type for this ? I think with this PR we will be using the same code in about 4 places

Reply:
Sure let me do it in a following refactor PR to unblock yours for now